lib/bpk-component-chip/stories.js

```diff
@@ -79,9 +79,32 @@ class StatefulBpkChipExample extends React.Component<
     };
   }
 
-  toggleChip = chip => {
+  toggleChip = (chip: 'flights' | 'hotels' | 'carHire' | 'trains') => {
```

Comment:
Changed this function to be the recommendation from flow regarding string literals:

https://flow.org/en/docs/types/literals/